### DISCUSSION TITLE
Sticky, floating windows infront of fullscreened windows

### DISCUSSION THREAD

**Discussion by mschwenzer** [i3/i3#4962] [Ideas]
I really would love to see  an option to **show** (at least: **_sticky_**) **floating windows in front of fullscreen windows**. I know there once was a short discussion on this: 
https://faq.i3wm.org/question/3482/all-floating-windows-in-front-of-fullscreen-one/index.html

The "problem" with the current behavior for me is that e.g. **edit panel windows** in a fullscreened program always are invisible until the fullscreen mode is exited (e.g. when adding a short text comment in a fullscreened Okular PDF-Reader) .
Another usecase are **quick entry windows** (e.g. yad/zenity). These never show up when being in fullscreen.

IMO this makes fullscreen windows less usable and the current behavior distracts the  workflow.  

Since there exist a lot of programs that in fact use editing panels and nonetheless  provide useful fullscreen modes (beyond increasing them by e.g. stacking) it would be nice to adapt i3 to allow for making good use of them.

**Comment by Airblader** [comment]
We won't be doing this. However, if https://github.com/i3/i3/issues/2107 gets implemented, it would emulate this closely enough, I think.

**Reply by mschwenzer** [reply]
Thanks a lot for the quick reply (and of course the great work that I really appreciate!). 

For me as far as I see this will not solve these problems since applications' fullscreen modes show different features than zoomed windows (and I actually want these application-specific features and also having the i3bar hidden when using fullscren windows). 

So I will try to search for information how i3 determines notifications (which actually _are_ treated as floating in fullscreen)  (and whether it is  possible to (externally) put regular windows into this "notification mode").
